### PR TITLE
Add rules_testing to bazelci.py

### PR DESCRIPTION
**Pull request by rickeylev**
Work towards ##1549

### CODE REVIEW

**Comment by rickeylev** [comment]
I think this part of the config is missing? I'm not familiar with these configs; just going off how I see rules_python and other rules configured.

**Comment by rickeylev** [comment]
Thanks, I couldn't find the url for its builds. I've got it setup to trigger on PRs now. I've filed https://github.com/bazelbuild/rules_testing/issues/11 to fix the breakages. I'll re-request review when our CI is green.

**Comment by rickeylev** [comment]
Ok, rules_testing passing at head. PTAL

File: buildkite/bazelci.py [hunk 1]
```diff
@@ -412,6 +412,12 @@
         "http_config": "https://raw.githubusercontent.com/bazelbuild/rules_swift/master/.bazelci/presubmit.yml",
         "pipeline_slug": "rules-swift-swift",
     },
+    "rules_testing": {
+        "git_repository": "https://github.com/bazelbuild/rules_testing.git",
+        "http_config": "https://raw.githubusercontent.com/bazelbuild/rules_python/master/.bazelci/presubmit.yml",
```
Comment (meteorcloudy) [inline buildkite/bazelci.py:417]:
```suggestion
        "http_config": "https://raw.githubusercontent.com/bazelbuild/rules_testing/master/.bazelci/presubmit.yml",
```

**Review by meteorcloudy — CHANGES_REQUESTED** [review]
LGTM, just needs to fix the rule name in the URL.

**Comment by rickeylev** [comment]
Fixed, PTAL

**Review by meteorcloudy — APPROVED** [review]

**Comment by meteorcloudy** [comment]
Started a build to update the last green commit for rules_testing: https://buildkite.com/bazel/rules-testing/builds/38

Tonight's nightly downstream run should be able to pick up rules_testing correctly.

**Pull request merged**